### PR TITLE
Improve verbose trace output

### PR DESCRIPTION
**Pull request by johnmccutchan**
@Hixie 

Related #8876

### CODE REVIEW

File: packages/flutter_tools/lib/src/devfs.dart [hunk 1]
```diff
@@ -339,14 +339,16 @@ class DevFS {
...
 
   Future<Uri> create() async {
+    printTrace('DevFS: Creating new filesystem on the device ($_baseUri)');
     _baseUri = await _operations.create(fsName);
     printTrace('DevFS: Created new filesystem on the device ($_baseUri)');
     return _baseUri;
   }
 
-  Future<dynamic> destroy() {
+  Future<dynamic> destroy() async {
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/devfs.dart]: `Future<Null>` ?

File: packages/flutter_tools/lib/src/resident_runner.dart [hunk 2]
```diff
@@ -226,7 +226,19 @@ abstract class ResidentRunner {
...
+        _serviceProtocolDone,
+        onError: _serviceProtocolError).whenComplete(appFinished);
+  }
+
+  Future<Null> _serviceProtocolDone(dynamic o) {
+    printTrace('Service protocol connection closed.');
+    return new Future<Null>.value(o);
+  }
+
+  Future<Null> _serviceProtocolError(dynamic e, StackTrace st) {
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/resident_runner.dart]: avoid abbreviations -- `object`, `error`, `stack`, etc.

**Comment by Hixie** [comment]
LGTM

File: packages/flutter_tools/lib/src/devfs.dart (hunk 1 above)
Comment (johnmccutchan) [inline packages/flutter_tools/lib/src/devfs.dart]: Done.

File: packages/flutter_tools/lib/src/resident_runner.dart (hunk 2 above)
Comment (johnmccutchan) [inline packages/flutter_tools/lib/src/resident_runner.dart]: Done.

**Pull request merged**